### ISSUE TITLE
Opensource version upgrade for 2023

### ISSUE THREAD

**Issue by anyj0527** [nnstreamer/nnstreamer#3976]
- ~~TensorFlow Lite v2.8.1 -> v2.11.0~~
  - Need to upgrade flatbuffers header (v2.0.0 -> v2.0.6)
    - tensorflow1-lite is not compatible with this new version
  - XNNPACK included build got failure in armv7l, armv7hl (It may need gcc version upgrade, https://github.com/tensorflow/tensorflow/issues/57364)
  - [tensorflow2 sandbox branch](https://review.tizen.org/gerrit/gitweb?p=platform%2Fupstream%2Ftensorflow2.git;a=shortlog;h=refs%2Fheads%2Fsandbox%2Fanyj0527%2Fv2.11.0): Upgraded to v2.11.0
  - [tensorflow1 sandbox branch](https://review.tizen.org/gerrit/gitweb?p=platform%2Fupstream%2Ftensorflow.git;a=shortlog;h=refs%2Fheads%2Fsandbox%2Fanyj0527%2Fv1.13.1-remove-system-flatbuf-dep): Detach the build dep to system flatbuffers header
  - toolchain:https://dashboard.tizen.org/index.code?sr=submit/tizen_base/20221218.050038 (will be applied by 12/19)

- Flatbuffers v2.0.0 -> ~~v22.11.23 (the latest)~~ v2.0.6 
  - ~~The header of v22.11.23 is compatible with TensorFLow Lite v2.11.0~~
  - https://review.tizen.org/gerrit/gitweb?p=platform%2Fupstream%2Fflatbuffers.git;a=shortlog;h=refs%2Fheads%2Fsandbox%2Fanyj0527%2Fv22.11.23
  - ~~nnfw (오형석님, 이성재님): Generated된 header를 그대로 사용 중 / 수정 완료~~: 
  - armnn (대인기님): Generated된 header를 그대로 사용 중 / 수정 필요

- ~~Protobuf v3.14.0 -> v3.20.3~~
  - ready to go!
  - https://review.tizen.org/gerrit/gitweb?p=platform%2Fupstream%2Fprotobuf.git;a=shortlog;h=refs%2Fheads%2Fsandbox%2Fsangjung%2F3.20.3
  - Error: platform/core/uifw/aurum (WIP - UIFW 김호상님)
  - /platform/upstream/pytorch (Done!)
  - Done!

- ~~orc v0.4.31 -> v0.4.33~~
  - In https://gstreamer.freedesktop.org/news/, It says the v0.4.33 supports aarch64 (64-bit ARM) architecture
  - Expect the performance improve in tensor_transform in aarch64 w/o fixing our source code.
  - Done!

- ~~openblas v0.3.7 -> 0.3.21~~
  -  ready to go!
  - https://review.tizen.org/gerrit/#/c/platform/upstream/openblas/+/284643/
  - Done

- Bazel (Ubuntu) v0.25.2 -> TBD (It depends on TF2)
  - Launchpad build failure for v0.25.2 has been fixed.
  - WiP

- TBU

**Comment by wooksong** [comment]
Do you guys have any plans for Meson?

**Comment by myungjoo** [comment]
We can ask Tizen-PM to upgrade meson.

**Comment by wooksong** [comment]
DLDT (OpenVino)' APIs we are currently using are deprecated.

**Comment by myungjoo** [comment]
> DLDT (OpenVino)' APIs we are currently using are deprecated.

That's also doable, but let's do it without announcing it via PM as we don't want any hard deadline or version declarations with it.

**Comment by again4you** [comment]
All packages are updated to `tizen.org` repositories so this issue is closed.
